cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -3041,6 +3107,11 @@ search = | tstats `cs_summariesonly_authentication` count from datamodel=Authent
 | table _time user dest src app count City Region Country pct \
 | `cs_authentication_successful_vpn_login_from_new_location_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = index=* `cs_vpn_indexes` tag=authentication action="success" dest_category="vpn_auth"
```

Comment:
Let's again discuss this, just in case if we can add filter here.

Reply:
**[Important]**
Please also create 1 Jira ticket to show only alerts on the **Overview page** which has "**action.cyences_notable_event_action = 1**".
Currently it's being determined by a macro, which is not properly being maintained and it's not a good idea anymore.

Reply:
> **[Important]** Please also create 1 Jira ticket to show only alerts on the **Overview page** which has "**action.cyences_notable_event_action = 1**". Currently it's being determined by a macro, which is not properly being maintained and it's not a good idea anymore.

Updated existing CY-414 Jira

Reply:
Updated search to limit the results